scipy/optimize/_highs/pyHiGHS/src/highs_wrapper.pyx

```diff
@@ -0,0 +1,622 @@
+                    - `5`: `SIMPLEX_SCALE_STRATEGY_HSOL`
+
+            - simplex_strategy : int {0, 1, 2, 3, 4}
+                Strategy for simplex solver. Default: 1. Corresponds
+                to the following:
+
+                    - `0`: `SIMPLEX_STRATEGY_MIN`
+                    - `1`: `SIMPLEX_STRATEGY_DUAL`
+                    - `2`: `SIMPLEX_STRATEGY_DUAL_TASKS`
+                    - `3`: `SIMPLEX_STRATEGY_DUAL_MULTI`
+                    - `4`: `SIMPLEX_STRATEGY_PRIMAL`
+
+            - simplex_update_limit : int
+                Limit on the number of simplex UPDATE operations.
+            - small_matrix_value : double
+                Lower limit on abs(matrix entries): values smaller
+                than this will be treated as zero.
+            - solution_file : str
+                Solution file
+            - solver : str {'simplex', 'ipm'}
+                Choose which solver to use.  If `solver='simplex'`
+                and `parallel=True` then PAMI will be used.
+            - time_limit : double
+                Max number of seconds to run the solver for.
+            - use_original_HFactor_logic : bool
+                Undocumented advanced option.
+            - write_solution_to_file : bool
+                Write the primal and dual solution to a file
+            - write_solution_pretty : bool
+                Write the primal and dual solution in a pretty
+                (human-readable) format
+
+        See [2]_ for a list of all non-advanced options.
+
+    Returns
+    -------
+    res : dict
+
+        If model_status is one of OPTIMAL,
+        REACHED_DUAL_OBJECTIVE_VALUE_UPPER_BOUND, REACHED_TIME_LIMIT,
+        REACHED_ITERATION_LIMIT:
+
+            - `status` : int
+                Model status code.
+            - `message` : str
+                Message corresponding to model status code.
+            - `x` : list
+                Solution variables.
+            - `slack` : list
+                Slack variables.
+            - `lambda` : list
+                Lagrange multipliers assoicated with the constraints
+                Ax = b.
+            - `s` : list
+                Lagrange multipliers associated with the constraints
+                x >= 0.
+            - `fun`
+                Final objective value.
+            - `simplex_nit` : int
+                Number of iterations accomplished by the simplex
+                solver.
+            - `ipm_nit` : int
+                Number of iterations accomplished by the interior-
+                point solver.
+
+        If model_status is not one of the above:
+
+            - `status` : int
+                Model status code.
+            - `message` : str
+                Message corresponding to model status code.
+
+    Notes
+    -----
+    If `options['write_solution_to_file']` is `True` but
+    `options['solution_file']` is unset or `''`, then the solution
+    will be printed to `stdout`.
```

Comment:
All single backticks in this paragraph need to be double backticks (which means "typeset as code").

Reply:
None of the cython docstrings are ever rendered.  Nonetheless, I have converted all single backticks to double back ticks